### PR TITLE
🧹  delete unused singer modules

### PR DESCRIPTION
**Pull request by sherifnada**
🧹 🧹 🧹 🧹 🧹

### CODE REVIEW

**Review by bazarnov — COMMENTED** [review]
Before we remove anything of those, we need to make sure the customers don't have any of listed connectors in use, kind of check of usage or something, also announce that such a change takes place, so everyone is aware of it and migrate their flow to the new native connectors.

File: airbyte-integrations/connectors/source-github/source_github/schemas/shared/user.json [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-      },
-      "received_events_url": {
-        "type": ["null", "string"]
-      },
-      "type": {
-        "type": ["null", "string"]
-      },
-      "site_admin": {
-        "type": ["null", "boolean"]
-      }
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-github/source_github/schemas/shared/user.json:57]: So, regards this change of source-github > schemas > shared > user.json - the file was just formated, right?

**Review by bazarnov — CHANGES_REQUESTED** [review]
LGTM, with small correction)

This leftover should be also removed.

 **SEED:**
- `airbyte-config/init/src/main/resources/seed/source_definitions.yaml` - Mixpanel-singer block should be removed

 **STANDARD_SOURCE_DEFINITION:**
- `airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/859e501d-2b67-471f-91bb-1c801414d28f.json` - Mixpanel-singer seed file.

 **.github > workflows > test-command.yml:**
 **.github > workflows > publish-command.yml:**
`https://github.com/airbytehq/airbyte/blob/822337136327a1ac80a635a40ad9022a75322e21/.github/workflows/test-command.yml#L91` this line is no longer needed, referencing to the singer-tap github.

**Comment by sherifnada** [comment]
@bazarnov thanks for the comments! made the changes you pointed out. Good catch!

**Pull request merged**